lib/blockstore/index.js

```diff
@@ -0,0 +1,42 @@
+/*!
+ * blockstore/index.js - bitcoin blockstore for bcoin
+ * Copyright (c) 2019, Braydon Fuller (MIT License).
+ * https://github.com/bcoin-org/bcoin
+ */
+
+'use strict';
+
+const {join} = require('path');
+
+const AbstractBlockStore = require('./abstract');
+const LevelBlockStore = require('./level');
+const FileBlockStore = require('./file');
+
+/**
+ * @module blockstore
+ */
+
+exports.create = (options) => {
+  const location = join(options.prefix, 'blocks');
+
+  if (options.memory) {
+    return new LevelBlockStore({
+      network: options.network,
+      logger: options.logger,
+      location: location,
+      cacheSize: options.cacheSize,
+      memory: options.memory
```

Comment:
Can you leave a comment that `LevelBockStore` is only used as an in memory database? It feels a bit obfuscated to check `if (options.memory)` and then pass along the variable without explicitly hardcoding in `memory: true` here

Reply:
Hmmm... I don't think this is necessary, and clear as it's within `if (options.memory)`.